tests/components/knx/test_scene.py

```diff
@@ -33,7 +30,7 @@ async def test_activate_knx_scene(hass: HomeAssistant, knx: KNXTestKit):
 
     registry = await async_get_entity_registry(hass)
     entity = registry.async_get("scene.test")
-    assert entity.entity_category == ENTITY_CATEGORY_DIAGNOSTIC
+    assert entity.entity_category == EntityCategory.DIAGNOSTIC
```

Comment:
Please use `is` for `entity.entity_category` comparison

Reply:
Agreed, done.